test/org/sagebionetworks/bridge/services/StudyServiceMockTest.java

```diff
@@ -631,6 +631,16 @@ public void loadingStudyWithoutAccountExistsSmsTemplateAddsADefault() {
         assertNotNull(retStudy.getAccountExistsSmsTemplate());
     }
     
+    @Test
+    public void loadingStudyWithoutSignedConsentSmsTemplateAddsADefault() {
```

Comment:
Should we also have the same tests for the email template?

Reply:
Verified we have create/load/update tests for a null email template